src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -76,6 +76,10 @@ services:
         class: stdClass
         decorates: decorated
         decoration_inner_name: decorated.pif-pouf
+    deprecated_service:
+        class: stdClass
+        deprecated: true
```

Comment:
Why do we need to define `deprecated` and `deprecated_template`? Why not do the same as for XML and just define `deprecated` with the message as the value?